libs/sdk-model/__mocks__/model.ts

```diff
@@ -59,6 +59,63 @@ export const WinRate: IMeasure<IMeasureDefinition> = newMeasure("aaX0PIUzg7nF");
  * Metric Type: MAQL Metric
  */
 export const Won: IMeasure<IMeasureDefinition> = newMeasure("afSEwRwdbMeQ");
+
+/**
+ * Creates an adhoc measure with aggregation "sum"
+ */
+export const AggregateAdhoc = (): IMeasure<IMeasureDefinition> => {
+    const adhocMeasure = newMeasure("afSEwRwdbMeQ");
+    adhocMeasure.measure.definition.measureDefinition.aggregation = "sum";
+    return adhocMeasure;
+};
+
+/**
+ * Creates an adhoc measure with empty filters
+ */
+export const EmptyFiltersAdhoc = (): IMeasure<IMeasureDefinition> => {
```

Comment:
a more 'native' approach to do this using model functions:

modifySimpleMeasure(Won, (m) => m.noFilters().defaultLocalId())

Reply:
Thanks, I've updated the PR.